examples/horizontal_convection.jl

```diff
@@ -108,7 +108,7 @@ simulation = Simulation(model, Δt=1e-2, stop_time=40.0)
 #
 # The `TimeStepWizard` manages the time-step adaptively, keeping the Courant-Freidrichs-Lewy 
 # (CFL) number close to `0.75` while ensuring the time-step does not increase beyond the 
-# maximum allowable value for numerical stability.
+# maximum allowable value for numerical stability (`new_Δt <= max_change * old_Δt`).
```

Comment:
```suggestion
# maximum allowable value for numerical stability (`new_Δt ≤ max_change * old_Δt`).
```

Reply:
Actually the phrasing here with the parenthesis introduced suggests that `max_change * old_Δt` is what numerical stability requires. But that's not true.